components/builder-web/app/shared/job-status-icon/job-status-icon.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input } from '@angular/core';
+import { iconForJobState, labelForJobState } from '../../util';
+
+@Component({
+  selector: 'hab-job-status-icon',
+  template: `<hab-icon [ngClass]="classes" [symbol]="symbol" [title]="label" [attr.title]="label"></hab-icon>`
+})
+export class JobStatusIconComponent {
+
+  @Input() job: any;
+  @Input() status: string;
+  @Input() animate: boolean = false;
+
+  private get _status() {
+    let s = '';
+
+    if (this.status) {
+      s = this.status;
+    } else if (this.job) {
+      s = this.job.state || '';
+    }
+
+    return s;
+  }
```

Comment:
could this be `return this.status || this.job ? this.job.state : '';` ?

Reply:
Totally could, yeah.